components/hab/src/cli.rs

```diff
@@ -295,6 +295,35 @@ pub fn get() -> App<'static, 'static> {
                     (@arg ORG: "The service organization")
                 )
             )
+            (@subcommand load =>
+                (about: "Load a service to be started and supervised by Habitat from a package or \
+                    artifact. Services started in this manner will persist through Supervisor \
+                    restarts.")
+                (@setting Hidden)
+            )
+            (@subcommand unload =>
+                (about: "Unload a persistent or transient service started by the Habitat \
+                    supervisor. If the Supervisor is running when the service is unloaded the \
+                    service will be stopped.")
+                (@setting Hidden)
+            )
+            (@subcommand start =>
+                (about: "Start a loaded, but stopped, Habitat service or a transient service from
```

Comment:
looks like we're missing a `\` at the end here which wraps the line in the terminal.